lib/layer-zero/README.md

```diff
@@ -0,0 +1,146 @@
+:warning: :bangbang: All examples under this folder are exploration code and have NOT been audited. Use them at your own risk! :bangbang: :warning:
```

Comment:
This message doesn't appear anywhere else in this repo - and it probably should!  At some higher level.  Perhaps multiple times?  (Maybe needs a separate issue if you agree?)

(Searched for "have not been audited" via github.)

Reply:
Yep, let's create a new issue.

Reply:
Just created one https://github.com/hashgraph/hedera-smart-contracts/issues/1217